interfaces/builtin/packagekit_control.go

```diff
@@ -46,30 +54,34 @@ dbus (receive, send)
         bus=system
         path=/org/freedesktop/PackageKit
         interface=org.freedesktop.PackageKit
-        peer=(label=unconfined),
+        peer=(label=###SLOT_SECURITY_TAGS###),
 dbus (receive, send)
         bus=system
         path=/org/freedesktop/PackageKit
         interface=org.freedesktop.PackageKit.Offline
-        peer=(label=unconfined),
+        peer=(label=###SLOT_SECURITY_TAGS###),
+
+# Read all properties from PackageKit
+# do not use peer=(label=unconfined) here since this is DBus activated
 dbus (send)
         bus=system
         path=/org/freedesktop/PackageKit
         interface=org.freedesktop.DBus.Properties
-        member=Get{,All}
-        peer=(label=unconfined),
+        member=Get{,All},
```

Comment:
As an alternative to removing the `peer=` attribute here, one another option would be to create a copy of the rule that uses `peer=(name=org.freedesktop.PackageKit)`.

I don't think we can combine these into a single rule to handle both the "service activation" case and "calling method on the unique name" case.